src/main/java/com/gmail/goosius/siegewar/settings/ConfigNodes.java

```diff
@@ -863,68 +857,62 @@
 			"",
 			"# If enabled, a firework will get spawned whenever a player dies inside a siege zone."),
 	PEACEFUL_TOWNS(
-			"peaceful_towns",
+			"neutral_towns",

```

Comment:
Changing this will mean that everyone's config's settings for peaceful towns will be lost, is this intentional?

Reply:
- Yes, I wanted to change the terminology in the config & lang files (but not the code)
- Its a good point though. I will add this onto the current migration guide for 0.7.x (and I'm looking at an automatic migration now also)